integration-tests/kafka-oauth-keycloak/pom.xml

```diff
@@ -44,6 +44,10 @@
             <groupId>io.strimzi</groupId>
             <artifactId>kafka-oauth-client</artifactId>
         </dependency>
+        <dependency>
```

Comment:
I believe we need to add it here to resolve the transitive, right?

Reply:
It's not pulled into runtime automatically by the kafka-client extension.